### PR TITLE
Rm2 possible fixes

### PR DESCRIPTION
**Pull request by harrylepotter**
couple of quick-n-dirty fixes to make this work on the RM2. Perhaps this could be put on a branch while we figure out a way to update wifi.go for dual RM1, RM2 support?

### CODE REVIEW

**Comment by Evidlo** [comment]
Can you split the Mercury News source into a separate PR?

**Comment by harrylepotter** [comment]
@Evidlo done. I think there might be more to the wifi fix though. Noticed the link changed (even without rebooting)

**Comment by Evidlo** [comment]
Hmm.  On rM1 its always the same

    dbus-monitor --system | grep /org/freedesktop/network1/link

I'm not sure if there's a nice way to get around this.  Dbus filters don't seem to have any wildcard functionality.

**Comment by limyewjin** [comment]
Repeating my suggestion here--What about changing match rule to use arg0:
"interface='org.freedesktop.DBus.Properties',member='PropertiesChanged',arg0='org.freedesktop.network1.Link'"

Does it work also in RM1? (looks like it should from your body dump in comments)

**Comment by Evidlo** [comment]
@limyewjin I just tried it and it works for rM1.  Did you test this on rM2?

	var rules = []string{
		"interface='org.freedesktop.DBus.Properties',arg0='org.freedesktop.network1.Link',member='PropertiesChanged'",
	}

@harrylepotter Can you change this in your PR?

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,6 @@ clean:
 	rm -f renews.x86 renews.arm release.zip
 
 define install
-	ssh-add
```
Comment (Evidlo) [inline Makefile:31]: Why are you removing this?

**Comment by harrylepotter** [comment]
Can confirm the rule change works. Added to PR

**Comment by harrylepotter** [comment]
ssh-add messes with my install at-least because i use a different ssh key. I'll put it back though..

**Comment by Evidlo** [comment]
@harrylepotter I wonder if there's a way to look up the path to the key for $(host).  It could be passed as an argument to ssh-add.

**Comment by harrylepotter** [comment]
@evidlo Possibly a cat/grep of ~/.ssh/config? Probably more trouble than
it’s worth though. I’m guessing most folks would just use the standard
~/.ssh/id_rsa ...

**Comment by Evidlo** [comment]
@harrylepotter Does this work for you?

```
define install
	eval $(shell ssh-agent -s)
	# stop running service, ignore failure to stop
	ssh -o AddKeysToAgent=yes root@$(host) systemctl stop renews || true
	...
```

**Pull request merged**